### ISSUE TITLE
FileChooser.accept not selecting the files

### ISSUE THREAD

**Issue by AlvaroMolinaBarquera** [puppeteer/puppeteer#5474]
#5473 # Steps to reproduce

**Tell us about your environment:**

* Puppeteer version: 2.1.1
* Node.js version: 12.14.1

**What steps will reproduce the problem?**

```typescript
    public static async uploadFile(page: puppeteer.Page, selector: string, path: string) {
        const [fileChooser] = await Promise.all([
            page.waitForFileChooser(),
            this.click(page, selector)
        ]);
        console.info(`Using the Path "${path}" on the current working directory "${process.cwd()}"`);
        await fileChooser.accept([path]);        
    }
```

**What is the expected result?**
- Select the file and triggering the file upload.

**What happens instead?**
- Nothing.
- Works properly on the version 2.0.0.

**Comment by playaroundHub** [comment]
I got the same issue
Puppeteer version 2.1.1
NodeJS version 12.14.0

**Comment by ruan-azevedo** [comment]
Same issue here with the method `elementHandle.uploadFile` Puppeteer v2.1.1, nothing happens.

Version 2.0.0 works fine

**Comment by dtboy1995** [comment]
same issue

**Comment by stack1ng** [comment]
same issue 
NodeJS v12.16.1 
puppeteer v2.1.1

**Comment by ghost** [comment]
The same problem here, the issue occurs in v2.1.0 and v2.1.1 as well.

NodeJS v13.6.0

**Comment by itsmrTech** [comment]
same issue
NodeJS v13.12.0
OS : Debian Stretch
puppeteer v2.1.1
I'm using puppeteer inside of a docker container with a base of this image: node:13.12.0-stretch

**Comment by paullewis** [comment]
This should be fixed as of 532ae573d29063ffd081ed9ee340d71b25320dec, so closing this 👍
